### PR TITLE
Issue #226 TotalPower SendHeader, sendData and stopData commands

### PR DESCRIPTION
**Pull request by LorenzoMonti**

### CODE REVIEW

File: simulators/totalpower/__init__.py [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
 class System(ListeningSystem):
 
     tail = [b'\x0A', b'\x0D']  # NEWLINE and CR
 
     ack = b'ack\n'
     nak = b'nak\n'
 
-    channels = 14
-
+    channels = 4 # changed for totalpower (instead of 14)
```
Comment (giuseppe-carboni) [inline simulators/totalpower/__init__.py:17]: Maybe is better to set this parameter dynamically inside the constructor, I'll handle this before merging

File: simulators/totalpower/__init__.py [hunk 2]
```diff
@@ -225,46 +231,91 @@ def _S(self, params):
...
-        pass
+        if len(params) == 1:
+    	      if params[0] == 0:
+    	          self.l_par = 0
+    	      else:
+                self.l_par = 1
 
     def _V(self, params):
-        pass
+        return "fpga 29.12.2009 firmware rev.48"
```
Comment (giuseppe-carboni) [inline simulators/totalpower/__init__.py:285]: Maybe it's better to move the firmware string inside a variable

**Review by LorenzoMonti — COMMENTED** [review]
the tests referring to TotalPower will be pushed shortly.

File: simulators/totalpower/__init__.py [hunk 3]
```diff
@@ -101,7 +102,7 @@ def parse(self, byte):
 
     def _execute(self, msg):
         msg = msg.strip()
-        print msg
+        print(msg)
```
Comment (giuseppe-carboni) [inline simulators/totalpower/__init__.py:105]: Prospector 1.2.0 gives me the following error regarding this line:
pylint: superfluous-parens / Unnecessary parens after 'print' keyword

File: simulators/totalpower/__init__.py [hunk 4]
```diff
@@ -74,6 +74,7 @@ def __init__(self, channels=14):
         self.externalNoise = 0
         self.toggle = 0
 
+        self.diagnostic_string = ''
```
Comment (giuseppe-carboni) [inline simulators/totalpower/__init__.py:77]: Saving this string as a class member seems unnecessary, the C command only serve diagnostic purposes

**Pull request merged**